yaml/examples/sample_daemonset.yaml

```diff
@@ -0,0 +1,35 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: iperf-client-daemonset
+  namespace: default
+  annotations:
+    kubernetes.io/description: |
+      This daemonset creates pods that uses sriov vfs.
+spec:
+  selector:
+    matchLabels:
+      name: iperf-client-daemonset
+  template:
+    metadata:
+      name: iperf-client-daemonset
+      labels:
+        name: iperf-client-daemonset
+        app: iperf
+      annotations:
+        v1.multus-cni.io/default-network: sriov-ovn-1
+    spec:
+#      nodeSelector:
+#            kubernetes.io/hostname: {nodeName}
+      containers:
+      - image: shaharklein/ub-gen:5
```

Comment:
can we create our image using Dockerfile and add it to the repo?

Reply:
sure, should there be any special specs for the image, other than it have iperf installed?

Reply:
no